types/japanese-holidays/japanese-holidays-tests.ts

```diff
@@ -1,4 +1,17 @@
-import { isHoliday, isHolidayAt, getHolidaysOf } from 'japanese-holidays';
+import {
+    getHolidaysOf,
+    getJDate,
+    getJDay,
+    getJFullYear,
+    getJHours,
+    getJMinutes,
+    getJMonth,
+    isHoliday,
+    isHolidayAt,
+    j2u,
+    jDate,
+    shiftDate,
```

Comment:
Why don't we add `u2j` and `uDate` tests?

Reply:
done, amended, thx!